site/content/extend/plugins/community_process.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Plugins on Community Server
+date: 2018-10-01T00:00:00-05:00
+subsection: Plugins (Beta)
+weight: 50
+---
+
+
+Getting your plugin onto community.mattermost.com is a valuable source of feedback. However we must ensure that our community server remains stable for everyone. This document outlines the process of getting your plugin onto the community server. In the future, some of these steps will be required to get your plugin into the integrations marketplace (TBD). When you are ready to have your plugin start this process, ask in the ~Toolkit channel on the community server.
+
+## Checklist
+
+### Deploying to ci-extensions
+- [Basic Code Review](#basic-code-review) passed
+- [CI system setup](#ci-system-setup) to build master
+- Has a [compatible licence](#compatible-licence).
+
+### Deploying to ci-extensions-release
+- Complete code review by two core committers. One focused on security.
+- [QA pass](#qa-pass)
+- [PM/UX review](#pm-ux-review)
+- Release created and [CI system setup](#ci-system-setup) to build releases
+
+### Deploying to community.mattermost.com
+- QA pass on ci-extensions-release of the release to deploy.
+
+
+## Step Definitions
+
+### Basic Code Review
+
+Basic code review of an experimental plugin involves a quick review by a [core committer](/contribute/getting-started/core-committers/) to verify that the plugin does what it says it does and to provide any guidance and feedback. To make it easier to provide feedback, a PR can be made that contains all the code of the plugin that isn't the boilerplate from mattermost-plugin-starter-template.
+
+### CI System Setup
+
+Setting up the CI system for your plugin will allow continuous testing of your master branch and releases on our testing servers. Master branch testing is done on https://ci-extensions.azure.k8s.mattermost.com/ and release testing is done on https://ci-extensions-release.azure.k8s.mattermost.com/
```

Comment:
Is setting up this CI possible for non mattermost plugins?

Reply:
Yes. The next paragraph explains that you will need to setup your own CI. Then the ci-extentions servers can pull from there. 

I would love suggestions on how to make this easy for people. I was thinking about making a circleci orb, but it seems quite difficult to get the latest artifact out of circleci.